### ISSUE TITLE
Develop multiple browser test strategy

### ISSUE THREAD

**Issue by yharich** [yellowhouseassociates/dashboard-ns#46]

**Comment by Teranode** [comment]
Normally i'd only test Mozilla & Chrome... I'd do internet explorer, but I've given up on it, maybe ill come back to it when Windows 10 releases.

**Comment by yharich** [comment]
Mozilla and Chrome are the browsers we focus on.  We've decided to ignore IE but if things happen to work in IE, great.  On the Mac/iOS side of things, we support Chrome on Mac.

But there are some occasions when Safari needs to be supported for the Phantomjs stuff which relies on the Webkit JS engine and there are some gotches between Webkit and V8 (The Chrome/Mozilla JS engine) that, as Mike discovered tonight, completely break the page.  Add to that some CSS issues because of course there is.

The blind spot I have is I don't pull up 2 different browsers and look at each change I make.  I want to but its a pain.

I worked before with Protractor http://angular.github.io/protractor/#/ and it will visit your pages in whichever browser you specify.  The list of browsers options are: android|chrome|firefox|htmlunit|internet explorer|iPhone|iPad|opera|safari

With this I can do my work and then run a  single command and sit back while I watch.  Having to manually exercise pages after each change would quickly become tedious but with this I can focus on looking for glaring presentation issues.
